app/experimenter/nimbus-ui/src/components/PageRequestReview/FormLaunchDraftToReview.tsx

```diff
@@ -0,0 +1,73 @@
+/* istanbul ignore file until EXP-866 final */
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React, { useCallback, useState } from "react";
+import Alert from "react-bootstrap/Alert";
+import Form from "react-bootstrap/Form";
+import { ReactComponent as AlertCircle } from "../../images/alert-circle.svg";
+import FormLaunchConfirmationCheckboxes from "./FormLaunchConfirmationCheckboxes";
+
+const FormLaunchDraftToReview = ({
+  isLoading,
+  onLaunchToPreview,
+  onSubmit,
+  onCancel,
+}: {
+  isLoading: boolean;
+  onLaunchToPreview: () => void;
+  onSubmit: () => void;
+  onCancel: () => void;
+}) => {
+  const [allBoxesChecked, setAllBoxesChecked] = useState(false);
+  const handleLaunchToPreviewClick = useCallback(
+    (ev) => {
+      ev.preventDefault();
+      onLaunchToPreview();
+    },
+    [onLaunchToPreview],
+  );
+
+  return (
+    <Alert variant="warning">
+      <Form className="text-body">
+        <p className="my-1">
+          <span className="text-danger">
+            <AlertCircle /> We recommend previewing before launch.
+          </span>{" "}
+          <a href="#" onClick={handleLaunchToPreviewClick}>
```

Comment:
When we move to development, semantically speaking this should be a `<button>` (but is correct semantically as a link now).

When I click on this in Storybook there's a really quick screen of "review status" first and then "preview status", but we can polish all this up when we work on this for real.

Reply:
I think the flash of "review status" is actually "submit success" - we do probably want to tweak that once this code actually starts submitting GQL queries